src/components/structures/MatrixChat.tsx

```diff
@@ -443,7 +443,9 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         if (crossSigningIsSetUp) {
             // if the user has previously set up cross-signing, verify this device so we can fetch the
             // private keys.
-            if (SecurityCustomisations.SHOW_ENCRYPTION_SETUP_UI === false) {
+
+            const cryptoExtension = ModuleRunner.instance.extensions.cryptoSetup;
+            if (cryptoExtension !== undefined && cryptoExtension.SHOW_ENCRYPTION_SETUP_UI == false) {
```

Comment:
is it *possible* for `cryptoExtension` to be undefined? `ExtensionManager.cryptoSetup` returns a `ProvideCryptoSetupExtensions`.

Reply:
I think the way the code evolved (with better encapsulation), that any given extension must always be defined. 

I will change call-sites etc accordingly.